installer/templates/new/widgets/text/text.scss

```diff
@@ -19,14 +19,20 @@ $moreinfo-color:    rgba(255, 255, 255, 0.7);
 
   .more-info {
     color: $moreinfo-color;
+    font-size: 24px;
```

Comment:
I like it more like it currently is:
![selection_074](https://cloud.githubusercontent.com/assets/645514/21585506/80910326-d0ca-11e6-9875-70d652d700cc.png)

Than how it looks with the changes introduced in this PR.
![selection_075](https://cloud.githubusercontent.com/assets/645514/21585507/847b4e42-d0ca-11e6-888a-3847b39c4201.png)

So let's not change it atm.